app/graph.py

```diff
@@ -276,6 +276,24 @@ def get_cluster_as(self):
                     if "--cluster-asn=" in arg:
                         asn = arg[14:]
                 logger.info('Kube-Router Detected! Cluster AS=%s',asn)
+        except Exception as e:
```

Comment:
The code was attempting to detect any CNI instead of exiting on the first one detected. This can cause issues if someone configures multiple CNI’s with different AS numbers. It would still be a problem but now its clear what AS is being used.